build/pack.c

```diff
@@ -348,6 +348,12 @@ static char *getIOFlags(Package pkg)
 	    compr = "lzma";
 	    (void) rpmlibNeedsFeature(pkg, "PayloadIsLzma", "4.4.6-1");
 #endif
+#if HAVE_ZSTD_H
+	} else if (rstreq(s+1, "zstdio")) {
+	    compr = "zstd";
+	    /* Add prereq on rpm version that understands zstd payloads */
+	    (void) rpmlibNeedsFeature(pkg, "PayloadIsZstd", "5.4.18-1");
```

Comment:
If you mean the tracking dependency, well, zstd got implemented elsewhere before this patch was submitted. Change and you, not me, will introduce confusion.